packages/cli/package.json

```diff
@@ -33,6 +33,7 @@
     "@react-native-community/cli-server-api": "^4.13.0",
     "@react-native-community/cli-tools": "^4.13.0",
     "@react-native-community/cli-types": "^4.10.1",
+    "appdirsjs": "^1.2.1",
```

Comment:
Adding a 17kB dependency just for that? How about using `os.tmpdir`?

Reply:
@thymikee

Apposite remark, but `os.tmpdir` has different behaviour. It returns a path to a system's cache directory and not a user's one.

Also, there is some logic to use the legacy path for backward compatibility.

Reply:
Yea, I don't really mind breaking this, it's only used for reminding a user that there's a new version of RN once in a while, nothing critical. I do appreciate your care about this though! 

Wonder if we really need a user directory _now_? Maybe it we cached more things, then I'd be on board with using this module. But now I can't justify using it. Also, a lot of tools use system's cache dir and it works just fine, although they usually make the path configurable.

Reply:
Alright, that's reasonable. I believe we do need a user's directory cause it is a standard de facto and we will need to store the cache there eventually. Yeah, YAGNI, but it's quite easy to use the user's cache directory.

So, I vote for storing cache there. If we don't want unnecessary dependencies, I can inline the code.

Reply:
I think for linux it'll be good enough to use `process.env.XDG_CACHE_HOME` and fallback to `~/.cache` to use the standard cache directory.